core/src/components/checkbox/checkbox.scss

```diff
@@ -20,12 +20,10 @@
    * @prop --transition: Transition of the checkbox icon
    *
    * @prop --checkmark-color: Color of the checkbox checkmark when checked
-   * @prop --checkmark-width: Stroke width of the checkbox checkmark
```

Comment:
Sorry I did not mean to remove the property here. This will remove it from the documentation. I just meant the definition below.